Client/mods/deathmatch/logic/CClientMarker.cpp

```diff
@@ -399,7 +399,7 @@ void CClientMarker::StreamOut()
 
 void CClientMarker::Callback_OnCollision(CClientColShape& Shape, CClientEntity& Entity)
 {
-    if (IS_PLAYER(&Entity))
+    if (IS_PLAYER(&Entity) && GetInterior() == Entity.GetInterior())            // Matching interior?
```

Comment:
For now, let's instead add the matching interior as an extra parameter to onClientMarkerHit call, just like matching dimension.